arenadata/Dockerfile.ubuntu.clang-check

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:22.04 AS base
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY README.Ubuntu.bash ./
+RUN set -eux && ./README.Ubuntu.bash
+RUN apt install -y rsync sudo python3.11 libipc-run-perl
+
+WORKDIR /home/gpadmin
+
+FROM base AS build
+
+COPY ./ gpdb_src
+
+WORKDIR /home/gpadmin/gpdb_src
+
+ENV CC="clang"
+ENV CXX="clang++"
+
+ENV CFLAGS="-O0 -Wno-deprecated-declarations"
+ENV CXXFLAGS="-O0 -Wno-deprecated-declarations"
+
+ENV CONFIGURE_FLAGS="--enable-gpcloud --enable-orafce --enable-tap-tests \
+                     --with-gssapi --with-libxml --with-openssl --with-perl \
+                     --with-python --with-uuid=e2fs --with-llvm --with-zstd \
+                     --enable-debug-extensions --enable-debug"
+
+RUN ./configure $CONFIGURE_FLAGS
+
+ENTRYPOINT make -s -j`nproc`
```

Comment:
What about running make at the previous step? In this case we will rid of `docker run`